lib/collection.js

```diff
@@ -184,6 +184,14 @@ function Collection(db, topology, dbName, name, pkFactory, options) {
   };
 }
 
+Object.defineProperty(Collection.prototype, 'optionsValidationLevel', {
```

Comment:
I <3 this.

Not for this PR, but I would like to see this for every inherited property

Reply:
me too!